sympy/polys/ring_series.py

```diff
@@ -101,18 +93,19 @@ def rs_is_puiseux(p, x):
     >>> rs_is_puiseux(p, x)
     True
     """
-    ii = p.ring.gens.index(x)
-    n = 1
+    index = p.ring.gens.index(x)
     for k in p:
-        if k[ii] != int(k[ii]):
+        if k[index] != int(k[index]):
             return True
-        if k[ii] < 0:
+        if k[index] < 0:
```

Comment:
The above change is semantic, not merely PEP8.
It's better to keep PEP8 corrections separate from semantic changes. People who're interested in a semantic change won't have to sift through a pile of PEP8 differences to find what they're after.